README.md

```diff
@@ -48,6 +48,7 @@ This list contains projects using OpenStreetMap data for creative purposes, as w
 
 * [iD](http://www.openstreetmap.org/edit?editor=id) - JavaScript based editor for the web browser with a wide array of presets. ([Source Code](https://github.com/openstreetmap/iD))
 * [Level0](http://level0.osmz.ru/) - Text based editor that's useful in specific corner cases. ([Source Code](https://github.com/zverik/level0))
+* [MapComplete](https://mapcomplete.osm.be) - an easy to use map viewer and editor, based on thematic maps. Mobile friendly, has simple questions and picture support.
```

Comment:
Please add new entries at the bottom of their respective categories.

Reply:
Ok, done